### PR TITLE
chore(components): fix outline tokens reference

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: libs/shared/src/lib/sass/mixins/appearance/_mixins.scss [hunk 1]
```diff
@@ -10,18 +10,6 @@
...
-  text: 'text',
-  ghost: 'ghost',
-);
-
-//$appearance-set: filled ghost outlined;
-//$states: idle disabled hover;
-
-
 @function get-variable($type) {
   $prefix: --_appearance-color;
```
Comment (rachelbt) [inline libs/shared/src/lib/sass/mixins/appearance/_mixins.scss:14]: :)

File: libs/shared/src/lib/sass/mixins/appearance/_variables.scss [hunk 2]
```diff
@@ -82,7 +95,12 @@ $states-mapping: (
       outline: transaprent,
     ),
     outlined: (
-      text: var(#{theme-consts.$vvd-color-neutral-30}),
+      text: var(#{theme-consts.$vvd-color-neutral-30}-strong),
```
Comment (rachelbt) [inline libs/shared/src/lib/sass/mixins/appearance/_variables.scss:98]: its disabled - not sure we need strong- not in outlined or duotone

File: libs/shared/src/lib/sass/mixins/appearance/_variables.scss [hunk 3]
```diff
@@ -74,26 +86,30 @@ $states-mapping: (
...
-      text: var(#{theme-consts.$vvd-color-neutral-30}),
+      text: var(#{theme-consts.$vvd-color-neutral-50}),
       fill: transaprent,
-      outline: var(#{theme-consts.$vvd-color-neutral-30}),
+      outline: var(#{theme-consts.$vvd-color-neutral-50}),
+    ),
+    duotone: (
+      text: var(#{theme-consts.$vvd-color-neutral-50}),
+      fill: transaprent,
+      outline: var(#{theme-consts.$vvd-color-neutral-50}-basis),
```
Comment (rachelbt) [inline libs/shared/src/lib/sass/mixins/appearance/_variables.scss:103]: I think the basis here is not needed for the variable :)

File: libs/shared/src/lib/sass/mixins/connotation/_variables.scss [hunk 4]
```diff
@@ -9,48 +9,55 @@ $compound-placeholder: '[compound]'; // will be replaced by actual compound
 $connotation-mapping: (
   // should map to colors e.g. purple-60
   primary: (
+    basis: theme-consts.$vvd-color-neutral,
     contrast: theme-consts.$vvd-color-neutral-90,
     fierce: theme-consts.$vvd-color-neutral-80,
-    strong: theme-consts.$vvd-color-neutral-90,
+    strong: theme-consts.$vvd-color-primary,
```
Comment (rachelbt) [inline libs/shared/src/lib/sass/mixins/connotation/_variables.scss:15]: this causes the button-primary-filled to have no hover color change

**Review by rachelbt — COMMENTED** [review]
I like the basis option :)
We should add it also to the colors I think.

**Comment by yinonov** [comment]
> I like the basis option :)
> 
> We should add it also to the colors I think.
> 
> 

U mean tokens? We discussed adding the actual 60 suffix

**Comment by rachelbt** [comment]
> > I like the basis option :)
> > We should add it also to the colors I think.
> 
> U mean tokens? We discussed adding the actual 60 suffix

I meant here:
https://vivid.deno.dev/designs/color
60 will be also great :)

File: libs/shared/src/lib/sass/mixins/appearance/_variables.scss (hunk 3 above)
Comment (yinonov) [reply libs/shared/src/lib/sass/mixins/appearance/_variables.scss:103]:
```suggestion
      outline: var(#{theme-consts.$vvd-color-neutral-50}),
```

File: libs/shared/src/lib/sass/mixins/connotation/_variables.scss (hunk 4 above)
Comment (yinonov) [reply libs/shared/src/lib/sass/mixins/connotation/_variables.scss]: introduced a new key instead

**Comment by yinonov** [comment]
@rachelbt I think we're already doing the renaming here. consider closing #343?

**Comment by yinonov** [comment]
> > > I like the basis option :)
> > > We should add it also to the colors I think.
> > 
> > 
> > U mean tokens? We discussed adding the actual 60 suffix
> 
> I meant here: https://vivid.deno.dev/designs/color 60 will be also great :)

oh in that case, it's not a new token coming from figma. it's a self mapping we use "under the hood", not an actual token

**Review by rachelbt — APPROVED** [review]
I think that the mapping is still not fully "there" yet, but we have a different PR for that. Apart from that - all look good to me.

**Pull request merged**